src/main/java/com/eternalcode/lobbyheads/position/Position.java

```diff
@@ -0,0 +1,116 @@
+package com.eternalcode.lobbyheads.position;
+
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Disclaimer - Bukkit {@link org.bukkit.Location} storage may cause a memory leak, because it is a wrapper for
+ * coordinates and {@link org.bukkit.World} reference. If you need to store location use {@link Position} and
+ * {@link PositionAdapter}.
+ * */
+public final class Position {
```

Comment:
As heads in Minecraft are basically blocks with additional features of rotation in X and Z axis, we can remove pitch. (I assume that Position is used purely for storing lobby heads)